src/adv.py

```diff
@@ -39,6 +41,11 @@
 
 # Make a new player object that is currently in the 'outside' room.
 
+player1 = Player("tommy", room["outside"])
+print(player1.current_room.name, player1.name)
+
+sword = Item('sword', 'A deadly weapon')
+print(f"__{sword.name}__:  {sword.description}")
```

Comment:
the printout of __sword__ makes me think I need to use "get __sword__", when it's just "get sword".